src/Fantomas.Core/CodePrinter.fs

```diff
@@ -2088,6 +2088,18 @@ and genExpr astContext synExpr ctx =
 
             expressionFitsOnRestOfLine short long
 
+        | DotGetAppWithLambda ((e, es, lpr, lambda, rpr, pr), lids) ->
+            leadingExpressionIsMultiline
+                (genAppWithLambda astContext sepNone (e, es, lpr, lambda, rpr, pr))
+                (fun isMultline ->
+                    if isMultline then
+                        (indent
+                         +> sepNln
+                         +> genSynLongIdent true lids
+                         +> unindent)
+                    else
+                        (indent >> genSynLongIdent true lids))
```

Comment:
This is dangerous that have an `indent` without later restoring using an `unindent`.

Reply:
All tests also seem to run just fine when removing `indent >>` so not sure if we still need that here.

Reply:
Mmh, yeah. I think that was just an artefact of me screwing around.